### PR TITLE
Completed css-grid-classes | Mohamed Elmessiry

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-01-21 at 9 46 52 PM

### CODE REVIEW

File: css-grid-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    </div>
+    <div class="row">
+      <div class="column-thrid">
+      <img src="../css-grid-classes/images/link.png" alt="warrior in action">
+      </div>
+      <div class="column-thrid">
+        <h2 >Super Smash Bros Ultimate</h2>
+        <p>Super Smash Bros. Ultimate received universal acclaim from both criticis and players, with some critics calling it the best installement in the series. It received praise for its large amount of content and fine tuning of existing smash gameplay elements. although its online mode was widely criticized</p>
+      </div>
+      <div class="column-thrid">
```
Comment (codingcodymiller) [inline css-grid-classes/index.html:22]: typo

File: css-grid-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+<body>
+  <div class="container">
+    <div class="row">
+      <h1>Choose your character!</h1>
+    </div>
+    <div class="row">
+      <div class="column-thrid">
+      <img src="../css-grid-classes/images/link.png" alt="warrior in action">
+      </div>
+      <div class="column-thrid">
```
Comment (codingcodymiller) [inline css-grid-classes/index.html:18]: typo

File: css-grid-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  <title>css-grid-classes</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <h1>Choose your character!</h1>
+    </div>
+    <div class="row">
+      <div class="column-thrid">
```
Comment (codingcodymiller) [inline css-grid-classes/index.html:15]: typo

File: css-grid-classes/style.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+}
+.row{
+  display:flex;
+  flex-direction: row;
+  align-content: center;
+}
+img{
+  width: 100%;
+}
+.column-thrid{
```
Comment (codingcodymiller) [inline css-grid-classes/style.css:16]: typo

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
You only have one grid class right now, supposedly for a column that is one third of the size available. This is only working right now because flex containers do not allow more than one row, so it is adjusting their size for you. You will need to add a column class for each size of column needed, do not let flexbox clean up your mess for you 😁

**Comment by Mohamed-Elmessiry** [comment]
> You only have one grid class right now, supposedly for a column that is one third of the size available. This is only working right now because flex containers do not allow more than one row, so it is adjusting their size for you. You will need to add a column class for each size of column needed, do not let flexbox clean up your mess for you 😁

Hi Cody, just to make sure I am getting this correctly. Do you need me to add a  a div, which would be a parent of <div class:column-third>, and a child to the <div class:row> ?

File: css-grid-classes/style.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  flex-direction: row;
+  align-content: center;
+}
+img{
+  width: 100%;
+}
+.column-third{
+  width: calc(100%/3)
+
+}
```
Comment (codingcodymiller) [inline css-grid-classes/style.css:19]: You're also going to need a `.column-fourth` and a `.column-two-thirds` classes. These will be used for the other sections with different widths.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Pull request closed without merging**